Sources/MapboxCoreNavigation/FreeDriveEventDetails.swift

```diff
@@ -0,0 +1,58 @@
+import Foundation
+import CoreLocation
+
+struct FreeDriveEventDetails: NavigationEventDetails {
```

Comment:
We avoid using the term “free drive” in API symbol names, in part because it’s too easy for the term to make something sound like an action method, but also because this jargon is unintuitive to developers who aren’t in the automotive industry. Instead, we always use the term “passive”, as in “passive navigation”. (Even the Android documentation has to quickly turn around and [define free-driving as passive navigation](https://docs.mapbox.com/android/navigation/guides/free-drive/).)

Reply:
Thanks, renamed all `FreeDrive` classes to the `PassiveNavigation`.